core/src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdStage.scala

```diff
@@ -428,7 +421,7 @@ import com.github.ghik.silencer.silent
             toSeqNr
         }
         session
-          .selectEventsByPersistenceId(persistenceId, pnr, expectedNextSeqNr, endNr, fetchSize)
+          .selectEventsByPersistenceId(persistenceId, pnr, expectedNextSeqNr, endNr)
```

Comment:
Where is the fetch size now from?

Reply:
The profile, i tried to remove everything from our config that was in the new driver config

Reply:
So we are using their default then I am guessing.

Reply:
The drivers? Yes